vllm/model_executor/layers/sampler.py

```diff
@@ -142,6 +146,22 @@ def _get_output_tokens(input_metadata: InputMetadata) -> List[List[int]]:
     return output_tokens
 
 
+def _apply_logits_processors(
+    input_metadata: InputMetadata,
+    logits: torch.Tensor,
+    output_tokens: List[List[int]]
+) -> torch.Tensor:
+    for _, seq_group in enumerate(input_metadata.seq_groups):
+        _, sampling_params = seq_group
+        logits_processors = sampling_params.logits_processors
+
+        if logits_processors is not None:
+            for logits_processor in logits_processors:
+                logits = logits_processor(logits, output_tokens)
```

Comment:
In this call, you send `output_tokens` to `logits_processor()`. However, in the `LogitsProcessor` interface, the `output_tokens` parameter does not exist:

```
def __call__(self, logits: torch.tensor) -> torch.tensor:
```

How does it work?